doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -327,7 +327,14 @@ Peripheral samples
 PMIC samples
 ------------
 
-|no_changes_yet_note|
+* Updated:
+
+  * The nPM1300: Fuel Gauge sample has been renamed to :ref:`npm13xx_fuel_gauge`.
```

Comment:
```suggestion
  * By renaming the nPM1300: Fuel Gauge sample to :ref:`npm13xx_fuel_gauge`.
```

Reply:
This reads better with "Updated", doesn't it?

Reply:
Understood. We have a bit different approach in the docs. We have the "headings" Added, Updated, Fixed and Removed. And the text coming after should read well with those.